configure.ac

```diff
@@ -1937,6 +1939,9 @@ AC_CONFIG_FILES([
   tools/tsxs
 ])
 
+AC_CONFIG_FILES([tools/check-unused-dependencies],
+                [chmod +x tools/check-unused-dependencies])
```

Comment:
After this change, Git owns tools/check-unused-dependencies.in. But that reminds me: I should add tools/check-unused-dependencies to .gitignore ...